dev/test/index.ts

```diff
@@ -1073,7 +1073,7 @@ describe('getAll() method', () => {
         expect(actualErrorAttempts).to.deep.eq(expectedErrorAttempts);
       });
     });
-  });
+  }).timeout(5000);
```

Comment:
Still needed?

Reply:
This is in master now, so the diff should eventually disappear: https://github.com/googleapis/nodejs-firestore/blob/master/dev/test/index.ts#L1103